contracts/contracts/RequestManager.sol

```diff
@@ -334,7 +348,7 @@ contract RequestManager is Ownable, LpWhitelist {
     /// @param requestId ID of the request.
     /// @param fillId The fill ID.
     /// @return The claim ID.
-    function claimRequest(uint256 requestId, bytes32 fillId)
+    function claimRequest(bytes32 requestId, bytes32 fillId)
```

Comment:
This type of thing would not be necessary if we used `uint256` instead of `bytes32` for the request ID, as mentioned above. I feel like treating the ID as an uint256 would just be nicer.

Reply:
see my comment above. I doesnt understand what "this type of thing" means

Reply:
> see my comment above. I doesnt understand what "this type of thing" means. If you mean changing the type, that's not a problem to my understanding. Any different representation, if wished, can be done outside of the protocol. We also don't convert other types to int just for nicer representation. We should do this on the representation level.

The thing we should ask ourselves is: how we use this ID? In what ways do we imagine it would be used?

The only reason this is `bytes32` is because the hash function returns that type. Semantically, this is an ID, as indicated by the name and previous typing we had. So, the way I am looking at it, our only usage is going to be for equality comparisons and lookups. There is no additional value provided by the data being interpreted as bytes here.

It is also not unusual at all to convert hash values to unsigned integers (e.g. just see Arbitrum contracts).